core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -733,4 +733,45 @@ package object config {
       .stringConf
       .toSequence
       .createWithDefault(Nil)
+
+  private[spark] val PYTHON_WORKER_REUSE = ConfigBuilder("spark.python.worker.reuse")
```

Comment:
I'd rather have separate files for these (e.g. `Python.scala`, `R.scala`) to avoid polluting this object. See other examples in the config package.

Reply:
Thanks for the nice suggestion. I was thinking config package object is already all-in-one (except `History`, `Kafka`, `Status`) so adding these config into here would be OK and we may have chance to sort out all the things, but I agree we can do this earlier for clear cases. Will address.